### ISSUE TITLE
Improve opening of a link in note editor via keyboard

### ISSUE THREAD

**Issue by abaevbog** [zotero/zotero#5199]
https://forums.zotero.org/discussion/122292/feature-request-allow-alt-enter-to-open-links-in-notes-for-accessibility

It is possible to open a link via keyboard but quite cumbersome and also not clear how if you rely on screen readers. One can navigate the popup that appears when the cursor is inside of a link by pressing Shift-Tab 3 times and then press Enter to open the link. The problem is there is no announcement when the popup appears, so if you rely on screen readers, you can only hope that it's there. Also, the popup does not appear when the cursor is right after the link - though screen readers will still announce that position as being on a link component.

So perhaps we should add an announcement that the popup is there, and have it appear when the cursor is right after the link too?

Maybe opening the link on Option/Ctrl+Enter keypress when the cursor is on the link is a nice addition as well? So that one does not have to navigate the popup to open it.

**Comment by abaevbog** [comment]
These 2 PRs add an announcement when the popup appears telling the user that the poup is open. I think this already would make a difference. 

We can then consider additional edits, like opening the link on Option/Ctrl+Enter or opening the popup when the cursor is right after (but still adjacent to) the link, if needed.
